demos/src/demo.scss

```diff
@@ -1,31 +1,29 @@
 $o-loading-is-silent: false;
 @import '../../main';
 
-h3 {
-	text-align: center;
-}
-
 .demo {
```

Comment:
Add demo classes to `body` using `origami.json` to avoid polluting the demo markup?